projects/lf-documentation/src/app/app.component.ts

```diff
@@ -107,19 +106,26 @@ export class AppComponent implements OnInit {
       language = navigator.language;
     }
     const langObj = {
-      'lf-localization-service-set-language': language
+      'lf-localization-service-set-language': language,
+    };
```

Comment:
should both 'lf-localization-service-set-language' and 'lf-localization-service-debug-mode' exist at the same time in the langObj?